aws/cloudformation/cloud_formation_stack.yml.erb

```
@@ -45,6 +66,19 @@ Parameters:
 <% end -%>
 Resources:
 
+  HourOfCodeCertificate:
+    Type: AWS::CertificateManager::Certificate
+    Properties:
+      DomainName: !Sub "www.${HourOfCodeSubdomainName}.${HourOfCodeBaseDomainName}"
+      SubjectAlternativeNames:
+        - !Sub "www.${HourOfCodeSubdomainName}.${HourOfCodeBaseDomainName}"
+      ValidationMethod: DNS
+      DomainValidationOptions:
+        - DomainName: !Sub "www.${HourOfCodeSubdomainName}.${HourOfCodeBaseDomainName}"
+          HostedZoneId: !Ref HourOfCodeHostedZoneId
+        - DomainName: !Sub "www.${HourOfCodeSubdomainName}.${HourOfCodeBaseDomainName}"
+          HostedZoneId: !Ref HourOfCodeHostedZoneId
```

I think you accidentally duplicated the www url and skipped the main url:

```suggestion
  HourOfCodeCertificate:
    Type: AWS::CertificateManager::Certificate
    Properties:
      DomainName: !Sub "${HourOfCodeSubdomainName}.${HourOfCodeBaseDomainName}"
      SubjectAlternativeNames:
        - !Sub "www.${HourOfCodeSubdomainName}.${HourOfCodeBaseDomainName}"
      ValidationMethod: DNS
      DomainValidationOptions:
        - DomainName: !Sub "${HourOfCodeSubdomainName}.${HourOfCodeBaseDomainName}"
          HostedZoneId: !Ref HourOfCodeHostedZoneId
        - DomainName: !Sub "www.${HourOfCodeSubdomainName}.${HourOfCodeBaseDomainName}"
          HostedZoneId: !Ref HourOfCodeHostedZoneId
```